style/pileup.css

```diff
@@ -138,11 +143,20 @@
   stroke-width: 2;
 }
 .pileup .network-status {
-  position: absolute;
-  font-size: small;
+  height: 100%;
```

Comment:
Can you include a screenshot of what this looks like? (I know, I know, we should have screenshot tests for pileup.)